example/output/json/components.examples/petExample.json

```diff
@@ -0,0 +1,13 @@
+{
+  "value": {
+    "id": 1,
+    "name": "Rover",
```

Comment:
even this is not a valid json schema. The type is missing

Reply:
In OpenAPI, examples don't have types, but this is likely an issue with `openapi-schema-to-json-schema` only supporting Schema Object and Parameter Object and supporting only OpenAPI 3.0. I believe `components.examples` is OpenAPI 3.1.